package.json

```diff
@@ -62,6 +62,7 @@
     "remark-rehype": "^8.0.0",
     "rss": "^1.2.2",
     "sequelize": "^6.6.5",
+    "sharp": "^0.30.7",
```

Comment:
when you ran that npm build command to fix `sharp` looks like you committed changes to updating sharp.

next time to fix those errors run:

```npm rebuild sharp --verbose```

can you uncommit the package.json and yarn.lock changes?

Reply:
It's asking me to send the local build to a different address. Should I try to run `npm rebuild sharp --verbose` to see if updating (?) that way will set everything correct? Gatsby logs say there are no errors, but is pinging something as deprecated.

Reply:
spoilers i did that ^ anyway and nothing resolved. i'll ping your team tomorrow e_e